packages/flutter_localizations/lib/src/l10n/material_en.arb

```diff
@@ -1,160 +1,142 @@
 {
   "scriptCategory": "English-like",
   "@scriptCategory": {
-    "description": "The name of the language's script category (see https://material.io/guidelines/style/typography.html#typography-language-categories-reference)",
-    "type": "text"
+    "description": "The name of the language's script category (see https://material.io/guidelines/style/typography.html#typography-language-categories-reference)"
   },
 
   "timeOfDayFormat": "h:mm a",
   "@timeOfDayFormat": {
-    "description": "The ICU 'Short Time' pattern, such as 'HH:mm', 'h:mm a', 'H:mm'. See: http://demo.icu-project.org/icu-bin/locexp?d_=en&_=en_US",
-    "type": "text"
+    "description": "The ICU 'Short Time' pattern, such as 'HH:mm', 'h:mm a', 'H:mm'. See: http://demo.icu-project.org/icu-bin/locexp?d_=en&_=en_US"
   },
 
   "openAppDrawerTooltip": "Open navigation menu",
   "@openAppDrawerTooltip": {
-    "description": "The tooltip for the leading AppBar menu (aka 'hamburger') button",
-    "type": "text"
+    "description": "The tooltip for the leading AppBar menu (aka 'hamburger') button"
   },
 
   "backButtonTooltip": "Back",
   "@backButtonTooltip": {
-    "description": "The BackButton's tooltip",
-    "type": "text"
+    "description": "The BackButton's tooltip"
   },
 
   "closeButtonTooltip": "Close",
   "@closeButtonTooltip": {
-    "description": "The CloseButton's tooltip",
-    "type": "text"
+    "description": "The CloseButton's tooltip"
   },
 
   "nextMonthTooltip": "Next month",
   "@nextMonthTooltip": {
-    "description": "The tooltip for the MonthPicker's 'next month' button.",
-    "type": "text"
+    "description": "The tooltip for the MonthPicker's 'next month' button."
   },
 
   "previousMonthTooltip": "Previous month",
   "@previousMonthTooltip": {
-    "description": "The tooltip for the MonthPicker's 'previous month' button.",
-    "type": "text"
+    "description": "The tooltip for the MonthPicker's 'previous month' button."
   },
 
   "nextPageTooltip": "Next page",
-  "@nextMonthTooltip": {
-    "description": "The tooltip for the [PaginatedDataTables]'s 'next page' button.",
-    "type": "text"
+  "@nextPageTooltip": {
```

Comment:
Do you think typos like that can be validated in `localizations_validator.dart`?

Reply:
Not by the current validator, since it's applied to the localizations that are available at runtime. We drop all of the @foo entries from the material_en.arb file, since they just provide meta-information for translators. 

That said, we should check it. I'll add validation to gen_localizations.dart in a separate PR.